test/e2e/playwright/tests/desktop/identifier_first/passkeys.login.spec.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2024 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+import { faker } from "@faker-js/faker"
+import { expect } from "@playwright/test"
+import { getSession } from "../../../actions/session"
+import { test } from "../../../fixtures"
+import { toConfig } from "../../../lib/helper"
+
+test.use({
+  addVirtualAuthenticator: true,
+})
+
+for (const mitigateEnumeration of [true, false]) {
+  test.describe(`account enumeration protection ${
+    mitigateEnumeration ? "on" : "off"
+  }`, () => {
+    test.use({
+      configOverride: toConfig({
+        mitigateEnumeration,
+        style: "identifier_first",
+        selfservice: {
+          methods: {
+            passkey: {
+              enabled: true,
+              config: {
+                rp: {
+                  display_name: "ORY",
+                  id: "localhost",
+                  origins: ["http://localhost:4455"],
+                },
+              },
+            },
+          },
+        },
+      }),
+    })
+
+    test("login", async ({ config, page, kratosPublicURL }) => {
+      const identifier =
+        await test.step("create webauthn identity", async () => {
+          await page.goto("/registration")
+          const identifier = faker.internet.email()
+          await page.locator(`input[name="traits.email"]`).fill(identifier)
+          await page
+            .locator(`input[name="traits.website"]`)
+            .fill(faker.internet.url())
+          await page.locator("button[name=method][value=profile]").click()
+
+          await page.locator("button[name=passkey_register_trigger]").click()
+
+          return identifier
+        })
+
+      await page.goto("/login")
+
```

Comment:
This test isn't doing anything besides registration?

Reply:
Passkeys autofills when you visit the page and submits the login automatically. This might feel unexpected for some users, though, so we might need to adjust this based on user fedback.

Reply:
Oh, there is no user interaction required where i click on the email field?

Reply:
I also added a test for the second step.